setup.py

```diff
@@ -91,9 +91,12 @@ def can_import(mname):
 Visit http://etetoolkit.org for more info.
 """
 
-extensions = [Extension('ete4.coretype.tree', ['ete4/coretype/tree.pyx']), 
-              Extension('ete4.smartview.draw', ['ete4/smartview/draw.pyx']), 
-              ],
+extensions = [
+        Extension('ete4.coretype.tree', ['ete4/coretype/tree.pyx']), 
+        Extension('ete4.parser.newick', ['ete4/parser/newick.pyx']), 
+        Extension('ete4.smartview.draw', ['ete4/smartview/ete/draw.pyx']), 
+        Extension('ete4.smartview.tree', ['ete4/smartview/ete/tree.pyx']), 
```

Comment:
This file has the Tree class commented out. I will remove it before merging... 
Maybe renaming the file? Now it has walk(), update_size() and a function called get_node()

Reply:
Oh, you mean the file `tree.pyx`! The functions walk() and update_size() could go elsewhere. Either way seems fine to me.